TestHttpsInsecure failures on nacl

Trying to vendor pprof for the next go release (1.9). The nacl trybots do not like the `TestHttpsInsecure` test.

```
--- FAIL: TestHttpsInsecure (0.00s)
	fetch_test.go:252: net.Listen: got error listen tcp: Protocol not available, want no error
FAIL
FAIL	cmd/vendor/github.com/google/pprof/internal/driver	0.556s
```

[Full log](https://storage.googleapis.com/go-build-log/b770f230/nacl-386_702218b9.log).

--- a/internal/driver/fetch_test.go
+++ b/internal/driver/fetch_test.go
@@ -241,6 +241,10 @@ func stubHTTPGet(source string, _ time.Duration) (*http.Response, error) {
 }
 
 func TestHttpsInsecure(t *testing.T) {
+	if runtime.GOOS == "nacl" {
+		t.Skip("test assumes tcp available")
+	}
+
 	baseVars := pprofVariables
 	pprofVariables = baseVars.makeCopy()
 	defer func() { pprofVariables = baseVars }()

